helm-chart/examples/gke-ephemeral/README.md

```
@@ -0,0 +1,474 @@
+tofu apply
+```
+
+Take the load balancer IP from the apply outputs:
+
+```bash
+tofu output -raw services_ingress_nginx_ip
+```
+
+Create two DNS records at your provider (Cloudflare, OVH, Route53, etc.):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)